src/cli/index.js

```diff
@@ -60,7 +60,8 @@ export function main({
     '--modules-folder <path>',
     'rather than installing modules into the node_modules folder relative to the cwd, output them here',
   );
-  commander.option('--cache-folder <path>', 'specify a custom folder to store the yarn cache');
+  commander.option('--preferred-cache-folder <path>', 'specify a custom folder to store the yarn cache if possible');
```

Comment:
Better to say "prefer a custom cache folder when it is available". We can make this change in a follow-up.